docker/dev/Dockerfile

```diff
@@ -28,7 +28,8 @@ RUN set -x && apt-get update -qq \
   libssl-dev \
   llvm-11 clang-11 llvm-11-dev libclang-11-dev \
   thrift-compiler libthrift-dev \
-  postgresql-server-dev-14 && \
+  postgresql-server-dev-14 \
+  postgresql-14 && \
```

Comment:
I do not know. These commits and their diffs above were generated automatically when I rebased this branch onto the newest master. I needed to rebase because the foundation for the metrics unit tests was in a newer commit that I did not have in my old master. I do not know about the exact contents of all the other commits that are now also merged as a result.

Reply:
After merging the current `master` into this branch, unrelated changes to this PR was removed from the diff correctly. This issue is resolved.